File: backend/model/aquarium_water_quality.py
class AquariumWaterQuality:
    def __init__(self, measured_at, ph, temperature, **kwargs):
        self.measured_at = measured_at
        self.ph = ph
        self.temperature = temperature


class AquariumWaterQualitySearchOption:
    def __init__(self, search_date, hour_from, hour_to, **kwargs):
        self.search_date = search_date
        self.hour_from = hour_from
        self.hour_to = hour_to

File: frontend/src/entity/aquarium-water-quality.ts
export default class AquariumWaterQuality {
    public measured_at: Date;
    public ph: number;
    public temperature: number;

    constructor(measured_at: Date, ph: number, temperature: number) {
        this.measured_at = measured_at;
        this.ph = ph;
        this.temperature = temperature;
    }
}
